OurUmbraco/Project/Api/ProjectUploadController.cs

```diff
@@ -24,6 +26,37 @@ public IEnumerable<WikiFile> GetProjectFiles()
             return files.Where(x => x.FileType == "package" || x.FileType == "hotfix").OrderByDescending(x => x.Version.Version).ToList();
         }
 
+        /// <summary>
+        /// Endpoint that takes a list of package file ids from UmbPack then archives them
+        /// </summary>
+        /// <param name="ids">Ids for package files that needs to be archived</param>
+        /// <returns></returns>
+        [HttpPost]
+        public HttpResponseMessage ArchiveProjectFiles(IEnumerable<int> ids)
+        {
+            var mediaProvider = new MediaProvider();
+
+            try
+            {
+                foreach (var archiveFileId in ids)
+                {
+                    var archiveFile = mediaProvider.GetFileById(archiveFileId);
+                    archiveFile.Archived = !archiveFile.Archived;
```

Comment:
Should `Archived` just be set to `true` rather than it toggling it's current value as it would be kind of weird behavior calling ArchiveProjectFiles and it un-archived a file?

Reply:
Good catch 🙂 Too quick to copy paste code 😅